### DISCUSSION TITLE
Pushing releases without changelog entry?

### DISCUSSION THREAD

**Discussion by SirRahikkala** [woocommerce/woocommerce#49511] [General]
Why was the 9.1.2 pushed to the world and there is no changelog entries for it? There is no good reason for it. If it was critical quick fix, we need to know what was it and if it was not so critical someone must have time to type few lines of text.

**Comment by ralucaStan** [comment]
Thank you for reaching out @SirRahikkala. You can read more about this release in [WooCommerce 9.1.2: Dot Release](https://developer.woocommerce.com/2024/07/12/woocommerce-9-1-2-dot-release/)
